examples/docs_snippets/docs_snippets/concepts/partitions_schedules_sensors/schedules/schedule_examples.py

```diff
@@ -9,11 +9,11 @@
 
 
 def test_configurable_job_schedule():
-    context = dg.build_schedule_context(
+    with dg.build_schedule_context(
```

Comment:
is it now necessary to use this as a context manager? what happens if you don't?

Reply:
it's only neccesary if you want to make a DagsterInstance.get() call in your sensor and have it use the instance you set on the context